### DISCUSSION TITLE
Individual SearchBar - Empty table

### DISCUSSION THREAD

**Discussion by tmigueis-belodigital** [filamentphp/filament#9946] [Ideas]
**When the table is empty, the individual searchbars don't appear, but I think it makes sense and is more practical to always appear.**

I made a change to the index file of the filament tables where I resolved this "problem", but as this view is in the vendor folder, when I make an update, this change will disappear, what can I do to prevent this change from disappearing?

**Comment by leandrocfe** [comment]
duplicated? https://github.com/filamentphp/filament/discussions/9947

**Comment by tmigueis-belodigital** [comment]
How?
